finvizfinance/util.py

```diff
@@ -82,6 +83,20 @@
 
 session = requests.Session()
 
+# Change User Agent on every call so it makes us more anonymous
+def get_header():
+    user_agents = [
```

Comment:
Does req header help? I think you are still using the same ip.

Reply:
Yes, you are right. If we create the session at the start. Then we don't need the different header. I have changed it like it was before. Thanks